tests/pkcli/build_data/1.in/db/000.yml

```diff
@@ -460,6 +460,15 @@ host:
           custom_image:
             docker_image: radiasoft/custom-image
             ssh_port: 3102
+          rsivizdevbox:
```

Comment:
I was a little confused by this. Maybe rsivizcoder?

Reply:
ff62898